scripts/backup.sh

```
@@ -1,7 +1,11 @@
 #!/bin/bash
 # scripts/backup.sh
+
+# Navegar para o diretório raiz do projeto
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)